app/models/cdo/manage/cdo.js

```diff
@@ -48,16 +50,14 @@ function ViewModel (tasklist, cdo, backNav, continueLink) {
             text: 'Dog name',
             classes: 'govuk-!-width-one-half'
           },
-          value: {
-            text: modelDetails.summary.dogName
-          }
+          value: { html: showValOrNotEntered(modelDetails.summary.dogName) }
```

Comment:
This potentially opens up script injection if the dog's name is something like '<script>alert(123)</script>' but the WAF will block. I'd advise against doing it this way if poss